pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/defaultvalues/RgpAutomaticProbandDataCreator.java

```diff
@@ -150,15 +149,20 @@ private static String getPhoneNumberFromActivities(List<Activities> activities)
         }).orElse("");
     }
 
-    void insertFamilyIdToDsmES(@NonNull String esIndex, @NonNull String participantId, long familyId) {
+    protected static void insertEsFamilyId(@NonNull String esIndex, @NonNull String ddpParticipantId, long familyId) {
         try {
-            Map<String, Object> esObjectMap = ElasticSearchUtil.getObjectsMap(esIndex, participantId, ESObjectConstants.DSM);
-            Map<String, Object> esDsmObjectMap = (Map<String, Object>) esObjectMap.get(ESObjectConstants.DSM);
-            esDsmObjectMap.put(ESObjectConstants.FAMILY_ID, familyId);
-            ElasticSearchUtil.updateRequest(participantId, esIndex, esObjectMap);
-            log.info("Family id for participant {} successfully added to ES", participantId);
+            Map<String, Object> esMap =
+                    ElasticSearchUtil.getObjectsMap(esIndex, ddpParticipantId, ESObjectConstants.DSM);
+            Map<String, Object> dsmMap = (Map<String, Object>) esMap.get(ESObjectConstants.DSM);
```

Comment:
if esMap.get(ESObjectConstants.DSM) is null, casting to Map might run int NPE, right ?
If thats never null, we might not need if (dsmMap == null) check